mergify_engine/service.py

```diff
@@ -22,8 +22,13 @@
 from mergify_engine import logs
 
 
+global SERVICE_NAME
+SERVICE_NAME: str = "engine-<unknown>"
+
+
 def setup(service_name: str, dump_config: bool = True) -> None:
-    service_name = "engine-" + service_name
+    global SERVICE_NAME
+    SERVICE_NAME = "engine-" + service_name
```

Comment:
why change the global variable? Is this used anything else?

Reply:
This is used to set the redis client name with the service name.